Neos.Eel/Classes/Helper/StringHelper.php

```diff
@@ -571,6 +571,26 @@ public function length($string)
     }
 
     /**
+     * Return numbers of words used in a string
+     *
+     * @param string $string The input string
+     * @return integer Length of the string
+     */
+    public function wordCount($unicode_string)
+    {
+        // First remove all the punctuation marks & digits
+        $unicode_string = preg_replace('/[[:punct:][:digit:]]/', '', $unicode_string);
+        // Now replace all the whitespaces (tabs, new lines, multiple spaces) by single space
+        $unicode_string = preg_replace('/[[:space:]]/', ' ', $unicode_string);
```

Comment:
Line comment are now cleaned up. About the functions: In line 582 I remove marks & digits and in line 584 I replace all kind of whitespaces with an space.